src/renderer/views/Watch/Watch.js

```diff
@@ -757,14 +761,21 @@ export default Vue.extend({
       if (this.rememberHistory && !this.isUpcoming && !this.isLoading && !this.isLive) {
         const player = this.$refs.videoPlayer.player
 
-        if (player !== null && this.saveWatchedProgress) {
+        if (player !== null && this.saveWatchedProgress && this.saveWatchedProgressForMusic) {
```

Comment:
If you want to save watch progress but not watch progress for music then wouldnt this not allow saving watch progress?

Reply:
Yes you are right, I fixed it now.